packages/components/src/Calendar/_docs/CalendarSingle.mdx

```diff
@@ -0,0 +1,28 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
```

Comment:
I've been co-locating the docs within their respective folders, so you'd have `src/Calendar/CalendarSingle/_docs/CalendarSingle.mdx`

Reply:
Ah yeah fair. It's because I was tossing up between a single `Calendars` doc or splitting them. Will move.